### ISSUE TITLE
cmd/upspinfs: causes noisy errors about failed SetTime on directories

### ISSUE THREAD

**Issue by masiulaniec** [upspin/upspin#470]
When using the common `rsync -a` operation to copy file trees to Upspin, `rsync` complains like so:

    rsync: failed to set times on "/u/<EMAIL>/testdir": Input/output error (5)

These low-value errors make it harder to spot real errors. I think it would be better if `upspinfs` faked success on these `SetTime` operations in the same manner it currently fakes `chmod` success.

**Comment by mmp** [comment]
Related, if I use `mv` to move a file to an upspinfs-mounted directory, I get:

```
mv: /Users/mpharr/up/mpharr@google.com/img.jpg: set times: No such file or directory
```

which is an extremely disconcerting error message. (Mac OSX, upspinfs as of a day or so ago.)

**Comment by masiulaniec** [comment]
The mv ENOENT problem got fixed in 860680cdc04e940ac98b8e333bc8fa033eb15f43.

The original problem with setting times on directories still stands.
